src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -596,7 +335,7 @@ private void ParseRequestLine(string requestLine, out string httpMethod, out Uri
         /// <param name="responseLine">The response line as a string.</param>
         /// <returns>The parsed status code from the response line.</returns>
         [SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic", Justification = "'this' is used when built in debug")]
-        private int ParseResponseLine(string responseLine)
+        internal int ParseResponseLine(string responseLine)

```

Comment:
>ParseResponseLine [](start = 21, length = 17)

Is this needed in the base class? It looks like it's only called by the ODataMultipartMixedBatchReader; can it be moved there?